<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size: 18px
}

QWidget{
	border: 1px solid;
	padding: 5px
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,2,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="text">
              <string>密碼:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pwdEdit">
             <property name="placeholderText">
              <string>(可選)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="text">
              <string>視頻類型:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="videoTypeCbx">
             <item>
              <property name="text">
               <string>mp4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mkv</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>隱寫類型:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="injectTypeCbx">
             <item>
              <property name="text">
               <string>File</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Folder</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="outputPathEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>請選擇輸出目錄</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outputPathSelBtn">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>選擇</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="coverVideoPathEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>請選擇作為載體的視頻</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="coverVideoSelBtn">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="text">
                <string>選擇</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="injectPathEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>請選擇要注入的文件/文件夾</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="injectPathSelBtn">
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="text">
                <string>選擇</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLineEdit" name="extractVideoPathEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>請選擇要提取的視頻</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="extractVideoSelBtn">
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="text">
              <string>選擇</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="startStegBtn">
           <property name="text">
            <string>開始隱寫</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="extractBtn">
           <property name="text">
            <string>提取目標</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,100">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>當前狀態：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="progressStateLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>無</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
